packetizer.go

```diff
@@ -68,5 +88,27 @@ func (p *packetizer) Packetize(payload []byte, samples uint32) []*Packet {
 	}
 	p.Timestamp += samples
 
+	if p.AbsSendTime != 0 {
+		t := toNtpTime(time.Now()) >> 14
```

Comment:
Go doesn't let you stub out time.Now() for tests. This code would be more testable if there was a way to override it here.

Reply:
How about a function object in the Packetizer structure? normally it'll call time.Now, but inside the package tests can override it?